### PR TITLE
updated route root->dashboard

### PR DESCRIPTION
**Pull request by aMaksimenko**
https://resolving.atlassian.net/browse/RES-68

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@
   resources :graphs
 
   resources :pages
-  resources :charts, only: [:index]
+  get 'charts/dashboard', to: 'charts#index'
```
Comment (Tensho) [inline config/routes.rb]: You have totally removed chart resource routes, I guess you mean something like this:
```
resources :charts, only: [:index] do
  resources : dashboards, only: [:index]
end
```

Comment (stasyswan) [inline config/routes.rb]: In this case we will have route like this
chart_dashboards GET    /charts/:chart_id/dashboards(.:format)  dashboards#index

And BTW we do not have Chart resource. I don't think we need resources routes here.

Comment (Tensho) [inline config/routes.rb]: We have `charts_controller`, is it correct that we don't need it any more?

Comment (stasyswan) [inline config/routes.rb]: No, we have charts_controller#index and need it.

**Pull request merged**